src/status_im/ui/screens/desktop/main/tabs/profile/views.cljs

```diff
@@ -31,7 +32,11 @@
      [react/text {:style           styles/profile-user-name
                   :font           :medium
                   :number-of-lines 1}
-      name])])
+      name])
+   (when-not (= name (gfy/generate-gfy public-key))
```

Comment:
not a big thing, `(when (not=` might be a bit better, though i'm puzzled with clojure's code style:

https://github.com/bbatsov/clojure-style-guide#not-equal (also https://github.com/bbatsov/clojure-style-guide#when-not but we don't have `(when (not` here :D )

Reply:
idk, should we add a PR to their repo to add this case?
I don't have any strong opinions on that one tho